pyt/cfg/stmt_visitor.py

```diff
@@ -1046,55 +1051,69 @@ def visit_Import(self, node):
                             retrieve_import_alias_mapping(node.names)
                         )
                     return self.add_module(
-                        module,
-                        name.name,
-                        name.asname,
-                        retrieve_import_alias_mapping(node.names)
+                        module=module,
+                        module_or_package_name=name.name,
+                        local_names=name.asname,
+                        import_alias_mapping=retrieve_import_alias_mapping(node.names)
                     )
         for alias in node.names:
+            # The module is uninspectable (so blackbox or built-in). If it has an alias, we remember
+            # the alias so we can do fully qualified name resolution for blackbox- and built-in trigger words
+            # e.g. we want a call to "os.system" be recognised, even if we do "import os as myos"
+            if alias.asname is not None and alias.asname != alias.name:
+                local_definitions = self.module_definitions_stack[-1]
+                local_definitions.import_alias_mapping[name.asname] = alias.name
             if alias.name not in uninspectable_modules:
-                log.warn("Cannot inspect module %s", alias.name)
+                log.warning("Cannot inspect module %s", alias.name)
                 uninspectable_modules.add(alias.name)  # Don't repeatedly warn about this
         return IgnoredNode()
 
     def visit_ImportFrom(self, node):
         # Is it relative?
         if node.level > 0:
             return self.handle_relative_import(node)
-        else:
```

Comment:
You're definitely improving the readability of this code, thank you :) The less indentation the better